apps/api/v2/src/ee/calendars/controllers/calendars.controller.ts

```diff
@@ -136,13 +136,14 @@ export class CalendarsController {
     @Req() req: Request,
     @Headers("Authorization") authorization: string,
     @Param("calendar") calendar: string,
-    @Query("redir") redir?: string | null
+    @Query("redir") redir?: string | null,
+    @Query("isDryRun") isDryRun?: string | null
```

Comment:
we should use `ParseBoolPipe` here and at all other places

```suggestion
    @Query("isDryRun", ParseBoolPipe) isDryRun?: boolean | null
```